src/nvim/screen.c

```diff
@@ -5409,7 +5409,7 @@ win_redr_custom (
       else
         stl = p_stl;
       use_sandbox = was_set_insecurely(
-          wp, (char_u *)"statusline", *wp->w_p_stl == NUL ? 0 : OPT_LOCAL);
+                                       wp, (char_u *)"statusline", *wp->w_p_stl == NUL ? 0 : OPT_LOCAL);
```

Comment:
at this point, the linebreak serves no point (tho cases like this might be simpler to handle with manual intervention).

Reply:
fixed.